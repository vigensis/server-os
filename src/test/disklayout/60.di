SCSI	disk00	MFG	DISK	600127266816	no	no
SCSI	disk01	MFG	DISK	600127266816	no	no
SCSI	disk02	MFG	DISK	600127266816	no	no
SCSI	disk03	MFG	DISK	600127266816	no	no
SCSI	disk04	MFG	DISK	600127266816	no	no
SCSI	disk05	MFG	DISK	600127266816	no	no
SCSI	disk06	MFG	DISK	600127266816	no	no
SCSI	disk07	MFG	DISK	600127266816	no	no
SCSI	disk08	MFG	DISK	600127266816	no	no
SCSI	disk09	MFG	DISK	600127266816	no	no
SCSI	disk0a	MFG	DISK	600127266816	no	no
SCSI	disk0b	MFG	DISK	600127266816	no	no
SCSI	disk0c	MFG	DISK	600127266816	no	no
SCSI	disk0d	MFG	DISK	600127266816	no	no
SCSI	disk0e	MFG	DISK	600127266816	no	no
SCSI	disk0f	MFG	DISK	600127266816	no	no
SCSI	disk10	MFG	DISK	600127266816	no	no
SCSI	disk11	MFG	DISK	600127266816	no	no
SCSI	disk12	MFG	DISK	600127266816	no	no
SCSI	disk13	MFG	DISK	600127266816	no	no
SCSI	disk14	MFG	DISK	600127266816	no	no
SCSI	disk15	MFG	DISK	600127266816	no	no
SCSI	disk16	MFG	DISK	600127266816	no	no
SCSI	disk17	MFG	DISK	600127266816	no	no
SCSI	disk18	MFG	DISK	600127266816	no	no
SCSI	disk19	MFG	DISK	600127266816	no	no
SCSI	disk1a	MFG	DISK	600127266816	no	no
SCSI	disk1b	MFG	DISK	600127266816	no	no
SCSI	disk1c	MFG	DISK	600127266816	no	no
SCSI	disk1d	MFG	DISK	600127266816	no	no
SCSI	disk1e	MFG	DISK	600127266816	no	no
SCSI	disk1f	MFG	DISK	600127266816	no	no
SCSI	disk20	MFG	DISK	600127266816	no	no
SCSI	disk21	MFG	DISK	600127266816	no	no
SCSI	disk22	MFG	DISK	600127266816	no	no
SCSI	disk23	MFG	DISK	600127266816	no	no
SCSI	disk24	MFG	DISK	600127266816	no	no
SCSI	disk25	MFG	DISK	600127266816	no	no
SCSI	disk26	MFG	DISK	600127266816	no	no
SCSI	disk27	MFG	DISK	600127266816	no	no
SCSI	disk28	MFG	DISK	600127266816	no	no
SCSI	disk29	MFG	DISK	600127266816	no	no
SCSI	disk2a	MFG	DISK	600127266816	no	no
SCSI	disk2b	MFG	DISK	600127266816	no	no
SCSI	disk2c	MFG	DISK	600127266816	no	no
SCSI	disk2d	MFG	DISK	600127266816	no	no
SCSI	disk2e	MFG	DISK	600127266816	no	no
SCSI	disk2f	MFG	DISK	600127266816	no	no
SCSI	disk30	MFG	DISK	600127266816	no	no
SCSI	disk31	MFG	DISK	600127266816	no	no
SCSI	disk32	MFG	DISK	600127266816	no	no
SCSI	disk33	MFG	DISK	600127266816	no	no
SCSI	disk34	MFG	DISK	600127266816	no	no
SCSI	disk35	MFG	DISK	600127266816	no	no
SCSI	disk36	MFG	DISK	600127266816	no	no
SCSI	disk37	MFG	DISK	600127266816	no	no
SCSI	disk38	MFG	DISK	600127266816	no	no
SCSI	disk39	MFG	DISK	600127266816	no	no
SCSI	disk3a	MFG	DISK	600127266816	no	no
SCSI	disk3b	MFG	DISK	600127266816	no	no
USB	key0	USBMFG	USB Key	15737028608	yes	no
