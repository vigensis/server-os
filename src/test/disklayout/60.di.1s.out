fatal error: invalid number of spares (1) for this disk configuration
